pkg/steps/script_step.go

```diff
@@ -163,9 +167,11 @@ tar zxvf %s.tgz 1>&2
 		}
 		var args []string
 		args = append(args, dockerArgs...)
+		args = append(args, "--entrypoint", c.Entrypoint)
```

Comment:
This seems to make it impossible to use the default entrypoint specified in the image. How about making this conditional, by adding `--entrypoint ENT` only when it isn't empty, like this?

```golang
if c. Entrypoint != "" {
    dockerArgs = append(dockerArgs, "--entrypoint", c. Entrypoint)
}
```